datacenter/dtr/2.2/guides/admin/configure/jobrunner.md

```diff
@@ -0,0 +1,261 @@
+is available to pick up the job, it will claim the it (i.e: the workerID will be set 
+to the replicaID of the jobrunner container that claimed the job) and set the job 
+status to `running`. The worker will carry out the job and then set the appropriate 
+status when complete (see [Job Statuses](#job-statuses) below).
+
+[//]: # (Placeholder for jobrunner scheduling. @sarahpark will work on the diagram)
+
+Each jobrunner has an internal queue of `waiting` jobs sorted by their `scheduledAt`
+time (from earliest to latest). When a worker is free to claim the next job, it claims
+it after a delay of up to 3 seconds. This delay is imposed so that each available worker
+has a chance to compete for the job. The worker that was successfully able to claim the
+job will set the job's `workerID` and all other workers will drop the job from their
+internal queue. If a job cannot be claimed due to capacity limits (see [Job Capacities](#job-capacities))
+then it is placed into a separate queue and will go through the claiming process
+when the worker has enough free capacity for it.
+
+Jobrunners monitor each other's `heartbeatExpiration` in the workers table. When a worker 
+see that another worker hasn't updated its expiration in a long time, it sets the 
+dead worker's status to `dead` and its jobs to `worker_dead`. If the dead worker is able to
+reconnect to the database and notices that it's jobs have been set to `worker_dead`,
+it sets those job statuses to `worker_resurrection` and cancels them.
+
+### Job Actions
+The available job actions are:
+
+- `gc`
+: Garbage collection deletes layers associated with deleted images.
+- `sleep`
+: Sleep is used to test the correctness of the jobrunner. It sleeps for 60 seconds.
+- `false`
+: False is used to test the correctness of the jobrunner. It runs the `false` command and immediately fails.
+- `tagmigration`
+- `bloblinkmigration`
+- `license_update`
+: License update checks for license expiration extensions if online license updates are enabled.
+- `nautilus_scan_check`
+: An image security scanning job. This job does not perform the actual scanning, rather it
+spawns `nautilus_scan_check_single` jobs (one for each layer in the image). Once
+all of the `nautilus_scan_check_single` jobs are complete, this job will terminate.
+- `nautilus_scan_check_single`
+: A security scanning job for a particular layer given by the `parameter: SHA256SUM`. This job
+breaks up the layer into components and checks each component for vulnerabilities
+(see [Security Scanning](../../user/manage-images/scan-images-for-vulnerabilities.md)).
+- `nautilus_update_db`
+: A job that is created to update DTR's vulnerability database. It uses an
+Internet connection to check for database updates through `https://dss-cve-updates.docker.com/` and
+updates the dtr-scanningstore container if there is a new update available (see [Set up vulnerability scans](set-up-vulnerability-scans.md)). 
+- `webhook`
```

Comment:
`webhook` is a job that is used to dispatch a webhook payload to a single endpoint